pyproject.toml

```diff
@@ -40,11 +40,11 @@ readme = 'README.md'  # Markdown files are supported
 include = [".git", "indica/git_version"]
 
 [tool.poetry.dependencies]
-python = "^3.8"
-prov = "^1.5.3"
+python = ">=3.8,<3.11"
```

Comment:
`scipy` started adding maximum python versions in 1.6.2, so `poetry` now complains if we use the `^` here

Reply:
This is an unnecessary restriction of poetry that I don't think `pip` has, every dependency version must be compatible with every python version.